presto-main/src/main/java/com/facebook/presto/type/DecimalInequalityOperators.java

```diff
@@ -124,64 +123,51 @@ private static SqlScalarFunction binaryOperator(OperatorType operatorType, Metho
                 .build();
     }
 
-    private static SqlScalarFunction binaryOperatorNullable(OperatorType operatorType, MethodHandle getResultMethodHandle)
+    @UsedByGeneratedCode
+    public static boolean opShortShort(long a, long b, MethodHandle getResultMethodHandle)
```

Comment:
I think methods were only removed from this class. Maybe it's easier to review the code side-by side instead